front/src/modules/simulationResult/components/SpaceTimeChart/useLazyProjectTrains.ts

```diff
@@ -21,27 +25,47 @@ const useLazyProjectTrains = ({
   path,
 }: UseLazyProjectTrainsOptions) => {
   const dispatch = useAppDispatch();
-  const loaderRef = useRef<TrainProjectionLazyLoader>(null);
+  const loaderRef = useRef<TrainProjectionLazyLoaderAbstract>(null);
   const timetableItemsByIdRef = useRef<Map<TimetableItemId, TimetableItem>>(new Map());
   const [projectedTrainsById, setProjectedTrainsById] = useState<
     Map<TimetableItemId, TrainSpaceTimeData>
   >(new Map());
+  const projectionType = useSelector(getProjectionType);
+
+  const { data: pathProperties } =
+    osrdEditoastApi.endpoints.postInfraByInfraIdPathProperties.useQuery(
+      {
+        infraId,
+        props: ['operational_points'],
+        pathPropertiesInput: {
+          track_section_ranges: path?.track_section_ranges || [],
+        },
+      },
+      {
+        skip: projectionType === 'trackProjection' || !path,
+      }
```

Comment:
You can use a skip token here, as we validated it at the last front meeting, and remove the `?` after path and the `[]` since you know it will be defined
```suggestion
       projectionType === 'operationalPointProjection' && path
        ? {
            infraId,
            props: ['operational_points'],
            pathPropertiesInput: {
              track_section_ranges: path.track_section_ranges,
            },
          }
        : skipToken
```

Reply:
Done